### PR TITLE
8254177: (tz) Upgrade time-zone data to tzdata2020b

### PR DESCRIPTION
**Pull request by kiranoracle**
Hi Guys,

Please review the patch for tzdata2020b integration into JDK.

Release details can be found here:

https://mm.icann.org/pipermail/tz-announce/2020-October/000059.html

Bug: https://bugs.openjdk.java.net/browse/JDK-8254177

I had the pacificnew, systemv files removed from JDK repo and so TimeZoneName classes, make file and few test files need to be updated to incorporate the change.

The patch has passed all the related testing including JCK.

Thanks,
Kiran

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8254177](https://bugs.openjdk.java.net/browse/JDK-8254177): (tz) Upgrade time-zone data to tzdata2020b

### Reviewers
 * [Erik Joelsson](https://openjdk.java.net/census#erikj) (@erikj79 - **Reviewer**)
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**)
 * [Sean Coffey](https://openjdk.java.net/census#coffeys) (@coffeys - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/602/head:pull/602`
`$ git checkout pull/602`

### CODE REVIEW

File: test/jdk/sun/util/calendar/zi/TestZoneInfo310.java [hunk 1]
```diff
@@ -215,8 +215,10 @@ public static void main(String[] args) throws Throwable {
 
         // test getAvailableIDs(raw);
         zids_new = TimeZone.getAvailableIDs(-8 * 60 * 60 * 1000);
-        //Arrays.sort(zids_new);
+        Arrays.sort(zids_new);
         zids_old = ZoneInfoOld.getAvailableIDs(-8 * 60 * 60 * 1000);
+	Arrays.sort(zids_old);
+	System.out.println("new and old length = " + zids_new.length + " " + zids_old.length);
```
Comment (kiranoracle) [inline test/jdk/sun/util/calendar/zi/TestZoneInfo310.java:221]: Missed to remove the instrumentation.

**Review by erikj79 — APPROVED** [review]
Build changes look good.

**Review by coffeys — APPROVED** [review]
Looks good. I think we should release-note the removal of the "US/Pacific-New" Link on the off chance that some production/testing system is looking for such a zone.

**Comment by kiranoracle** [comment]
Thanks for the review everyone, I have added a release note to the bug. I will integrate the changes

**Comment by gnu-andrew** [comment]
Looks like we both reached the same patch for this one independently :)
systemv was removed from tzdata some time ago and wasn't even being read by the JDK as far as I could see. Its contents are duplicated in jdk11_backward anyway.
Good to see both it and pacificnew being removed. In the unlikely event the removal of Pacific_New is problematic, it could be added to jdk11_backward as well, but it seems very unlikely.

**Comment by kiranoracle** [comment]
Thanks for the comment, Andrew. I agree

**Pull request closed without merging**